kuksa_databroker/integration_test/gen_proto/sdv/databroker/v1/broker_pb2.pyi

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
What is the reason for including these generated files? If there is a practical need to have them included it could maybe be an idea to write a few words on how/when they shall be re-generated.

Reply:
For the `_pb2*.py` files, it's because the integration tests should be able to verify that the external interface doesn't break when updating the `.proto`-files. If we want to make breaking changes, updating the generated files should be done manually.

I added the script for generating the `proto`-files.